### DISCUSSION TITLE
Lapor contoh kode kurang lengkap pada dokumentasi

### DISCUSSION THREAD

**Discussion by morizbebenk** [kirimwa-id/forum#25] [Umum]
Halo, sepertinya contoh kode kurang lengkap pada dokumentasi bagian [ini](https://developer.kirimwa.id/api/v1/?php#get-messages-message_id)

```
try {
  $reqParams = [
    'token' => getenv('API_TOKEN'),
    'url' => sprintf('https://api.kirimwa.id/v1/messages/%s', getenv('MESSAGE_ID'))
  ];

  $response = apiKirimWaRequest($reqParams);
  echo $response['body'];
} catch (Exception $e) {
  print_r
```

Terimakasih,

**Comment by rioastamal** [comment]
Halo @morizbebenk terima kasih laporannya. Sekarang dokumentasi [PHP untuk MESSAGE_ID](https://developer.kirimwa.id/api/v1/?php#get-messages-message_id) sudah diperbaiki.
